src/components/product_filter/ProductFilterSub.jsx

```diff
@@ -0,0 +1,52 @@
+import "./ProductFilter.css";
+import { RadioInput } from "../filter_input/RadioInput";
+import { CheckboxInput } from "../filter_input/CheckboxInput";
+import { PriceRangeInput } from "../filter_input/PriceRangeInput";
+const ProductFilterSub = () => {
+	const categoryOptions = [
```

Comment:
Suggestion: can abstract data in other files

Reply:
This is dummy data. On adding functionality this data will be comping from API & context.